webapp/store/propTypes/pluginMetadata.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
```

Comment:
It's not uncommon for a PropType structure to be used in multiple places (especially if you're connected to the state). Rather than export from the first file that uses it and import in the others, I decided it might help to create a dedicated place for them. I haven't come across any sort of widely used convention for where to keep these, so I thought that a folder in the store with the redux modules might be good. Even though this doesn't have anything to do with redux I thought it might make sense since this when it's most used. Alternatively, maybe having this propTypes folder one level up and next to `components` and `containers` might make more sense though. Open to suggestions on this one before committing!